src/backends/gr.jl

```diff
@@ -750,7 +751,7 @@ function gr_display(sp::Subplot{GRBackend}, w, h, viewport_canvas)
             isfinite(clims[1]) && (zmin = clims[1])
             isfinite(clims[2]) && (zmax = clims[2])
         end
-        GR.setspace(zmin, zmax, 35, 60)
+        GR.setspace(zmin, zmax, map(Int,map(round,sp[:camera]))...)
```

Comment:
Any particular reason for not `round.(Int, sp[:camera])...`? That's somewhat clearer in my view.

Reply:
Mainly my lack off full familiarity with Julia... Your way is clearer to me will change.